science/applications/app.cpp

```diff
@@ -40,13 +39,16 @@ hal::status application(science::hardware_map &p_map) {
     int revolver_hall_value = 1;
     int seal_hall_value = 1;
 
+    // Constants
+    const float MIN_SEAL_DUTY_CYCLE = 0.065f;
+    const float MAX_SEAL_DUTY_CYCLE = 0.085f;
```

Comment:
```suggestion
    static constexpr float MIN_SEAL_DUTY_CYCLE = 0.065f;
    static constexpr float MAX_SEAL_DUTY_CYCLE = 0.085f;
```

Prefer to make constants compile time and static, in general.